kakao-shop/src/components/atoms/Footer.jsx

```diff
@@ -1,19 +1,25 @@
+import MainContainer from "./MainContainer";
+
 /**
  * 전체 페이지에 공통으로 들어가는 footer
  */
 const Footer = () => {
   return (
     <footer>
-      <p className="whitespace-pre-wrap mx-auto px-12 py-10 text-xs text-gray-400">
-        (통화료 발생 / 평일 09:00~18:00) 톡상담하기 (평일 09:00~18:00)
-        <br />
-        Copyright © kakao-tech-campus-SJH. All rights reserved
-        <br />
-        (주)카카오에서 판매하는 상품 중에는 개별판매자가 판매하는 상품이 포함되어 있습니다.
-        <br />
-        개별판매자가 판매하는 상품에 대해 (주)카카오는 통신중개판매업자로서 통신판매의 당사자가 아니며 상품의 주문, 배송
-        및 환불 등과 관련한 의무와 책임은 각 판매자에게 있습니다.
-      </p>
+      <MainContainer className="">
```

Comment:
```suggestion
      <MainContainer >
```
빈문자열을 props로 넘겨주는게 어색하게 느껴지네요 default값 처리를 하고 상위에서는 안 넘겨주는게 어떨까요?